rainbow/simulators/prox_rigid_bodies/api.py

```diff
@@ -18,6 +18,7 @@
 from rainbow.simulators.prox_rigid_bodies.types import *
 import rainbow.math.matrix3 as M3
 
+unique_name_counter = 0
```

Comment:
Why this change to this function?

Is this thread safe, module variable? what if multiple imports are used in different modules?